.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -187,7 +190,10 @@ jobs:
       uses: Azure/powershell@38b0aa56c413239bd9dcc898d1f62986233658bc
       with:
         inlineScript: |
-          Stop-AzWebAppSlot -ResourceGroupName ${{ inputs.resource-group-name }} -Name ${{ inputs.app-name }} -Slot ${{ inputs.slot-name }}
+          Stop-AzureWebAppSlot `
```

Comment:
This and `Start-AzureWebAppSlot` below are custom scripts, right? Because then these steps should rather just use `run` with `pwsh`. Same in `reset-environment`, also for all the other steps.

Reply:
Yes they are custom scripts, but if I use the default `run` and `pwsh` then I need to add an extra step, see: https://lombiq.atlassian.net/browse/NEST-338?focusedCommentId=71347

Reply:
If you mean that "Install and import modules" then isn't it supposed to happen with _Lombiq/Infrastructure-Scripts/.github/actions/initialize_?

Reply:
_Lombiq/Infrastructure-Scripts/.github/actions/initialize_ copies the modules, so they will be usable in this repository, but it doesn't installs them. I will try to add that step to it.